MemorialHouse/MHPresentation/MHPresentation/Source/Extensions/UILabel+Style.swift

```diff
@@ -0,0 +1,37 @@
+import UIKit
+
+enum LabelStyle {
+    case `default`
+    case header
+    case body
+}
+
+extension UILabel {
+    convenience init(style: LabelStyle) {
+        self.init(frame: .zero)
+        self.translatesAutoresizingMaskIntoConstraints = false
+        switch style {
+        case .default: configureDefault()
+        case .header: configureHeader()
+        case .body: configureBody()
+        }
+    }
+    
+    func configureDefault() {
+        self.font = UIFont.ownglyphBerry(size: 25)
+        self.textColor = .title
+        self.textAlignment = .center
+    }
+    
+    func configureHeader() {
+        self.font = UIFont.ownglyphBerry(size: 30)
+        self.textColor = .title
+        self.textAlignment = .center
+    }
+    
+    func configureBody() {
+        self.font = UIFont.ownglyphBerry(size: 17)
+        self.textColor = .title
+        self.textAlignment = .center
+    }
```

Comment:
P1: textColor와 textAlignment는 모두 공통되는 것 같은데 아래와 같이 수정할 수 있을 것 같습니다!

```suggestion
self.textColor = .title
self.textAlignment = .center
switch style {
    case .default: 
        self.font = UIFont.ownglyphBerry(size: 25)
    case .header: 
        self.font = UIFont.ownglyphBerry(size: 30)
    case .body:
        self.font = UIFont.ownglyphBerry(size: 17)
}
```

꼭 이렇게가 아니어도 `textColor`와 `textAlignment`는 한 번에 설정해줄 수 있지 않을까요??

Reply:
헉 멍청했네요 맞습니다...!!!! 반영하겠습니다